lti_consumer/plugin/views.py

```diff
@@ -226,11 +226,16 @@ def deep_linking_response_endpoint(request, lti_config_id=None):
 
     # If LtiConfiguration doesn't exist, error with 404 status.
     except LtiConfiguration.DoesNotExist as exc:
-        log.info("LtiConfiguration does not exist.")
+        log.info("LtiConfiguration %s does not exist: %r", lti_config_id, exc)
         raise Http404 from exc
     # Bad JWT message, invalid token, or any message validation issues
-    except (Lti1p3Exception, PermissionDenied):
-        log.info("Permission denied for user.")
+    except (Lti1p3Exception, PermissionDenied) as exc:
+        log.warning(
+            "Permission on LTI Config %s denied for user %s: %r",
+            lti_config,
```

Comment:
It's a little tricky using `lti_config` here, since its assignment is inside the try block, so in theory this clause could be triggered before the variable has a value.  But I think it is OK in this case.

Reply:
@nedbat `Lti1p3Exception` and `PermissionDenied` will only ever happen after the lti configuration is retrieved. The only other case where this variable isn't set is covered by `LtiConfiguration.DoesNotExist`